app/_locales/de/messages.json

```diff
@@ -727,9 +727,6 @@
   "customGasSubTitle": {
     "message": "Höhere Gebühren können Bearbeitungszeiten verkürzen, wofür es allerdings keine Garantie gibt."
   },
-  "customNetworks": {
```

Comment:
... odd :thinking: This was added in the last crowdin PR, now it is being removed.

Reply:
Oh I see. The `en` message was deleted but not the translations. Merge error, this is why `develop` is currently failing.